### PR TITLE
feat/PRO-2408/highlighted-media-grid

### PR DESCRIPTION
**Pull request by RanaBug**
## Description

- UI and API integration of the Highlighted Media Collection Grid Tile

## How Has This Been Tested?

- Unit tests and manual testing

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few small items but looks great and can't wait to see this in action! 🚀

File: src/apps/pillarx-app/components/HighlightedMediaGridTile/HighlightedMediaGridTile.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+const HighlightedMediaGridTile = ({
+  data,
+  isDataLoading,
+}: HighlightedMediaGridTileProps) => {
+  const [mediaGridTileWidth, setMediaGridTileWidth] = useState<number>(0);
+  const { data: dataMedia, meta } = data || {};
+  const dataMediaGrid = dataMedia as MediaGridData | undefined;
+
+  // This mesure the window width
```
[the remark this answers is not in the document]
Comment (RanaBug) [reply src/apps/pillarx-app/components/HighlightedMediaGridTile/HighlightedMediaGridTile.tsx:26]: Oops, I wrote it in French haha

File: src/apps/pillarx-app/components/HighlightedMediaGridTile/HighlightedMediaGridTile.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        </div>
+      </TileContainer>
+    );
+  }
+
+  if (!dataMediaGrid?.grids) return null;
+
+  return (
+    <div ref={divRef}>
+      <TileContainer
```
[the remark this answers is not in the document]
Comment (RanaBug) [reply src/apps/pillarx-app/components/HighlightedMediaGridTile/HighlightedMediaGridTile.tsx:85]: Not here, we have an id for the E2E testing, however I did not need a data-testid for the jest unit test.

File: src/apps/pillarx-app/components/MediaGridCollection/DisplayCollectionImage.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+    return (
+      <div
+        id="pillarx-feed-display-collection-avatar"
+        data-testid="display-collection-image"
+        className={`overflow-hidden ${className} ${url && 'cursor-pointer'}`}
+        onClick={handleClick}
+      >
+        <Avatar
+          name="Maria Mitchell"
```
[the remark this answers is not in the document]
Comment (RanaBug) [reply src/apps/pillarx-app/components/MediaGridCollection/DisplayCollectionImage.tsx:49]: My mistake, I thought I changed that

File: src/apps/pillarx-app/components/HighlightedMediaGridTile/HighlightedMediaGridTile.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (IAmKio) [reply src/apps/pillarx-app/components/HighlightedMediaGridTile/HighlightedMediaGridTile.tsx:85]: Got it 👍

**Review by IAmKio — APPROVED** [review]
LGTM! 🚀

**Pull request merged**